packages/rich-text-structure/package.json

```diff
@@ -0,0 +1,27 @@
+{
+	"name": "@wordpress/rich-text-structure",
+	"version": "1.0.0",
+	"description": "Rich text structure and manipulation API.",
+	"author": "The WordPress Contributors",
+	"license": "GPL-2.0-or-later",
+	"keywords": [
+		"wordpress",
+		"rich-text"
+	],
+	"homepage": "https://github.com/WordPress/gutenberg/tree/master/packages/autop/README.md",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/WordPress/gutenberg.git"
+	},
+	"bugs": {
+		"url": "https://github.com/WordPress/gutenberg/issues"
+	},
+	"main": "build/index.js",
+	"module": "build-module/index.js",
+	"dependencies": {
+		"@babel/runtime-corejs2": "7.0.0-beta.56"
```

Comment:
I saw that `lodash` is one of the dependencies. We should list all other deps if it uses them, too.

Reply:
Thanks, forgot it's a separate package entirely.